introduction.md

```diff
@@ -1,4 +1,10 @@
 I am Akshay. I love courses on CodeSchool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?